src/bun.js/uuid.zig

```diff
@@ -129,3 +134,76 @@ pub const zero: UUID = .{ .bytes = .{0} ** 16 };
 pub fn newV4() UUID {
     return UUID.init();
 }
+
+/// # --- 48 ---   -- 4 --   - 12 -   -- 2 --   - 62 -
+/// # unix_ts_ms | version | rand_a | variant | rand_b
+pub const UUID7 = struct {
+    bytes: [16]u8,
+
+    var uuid_v7_lock = bun.Lock{};
+    var uuid_v7_last_timestamp: std.atomic.Value(u64) = .{ .raw = 0 };
+    var uuid_v7_counter: std.atomic.Value(u32) = .{ .raw = 0 };
+
+    fn getCount(timestamp: u64) u32 {
+        uuid_v7_lock.lock();
+        defer uuid_v7_lock.unlock();
+        if (uuid_v7_last_timestamp.swap(timestamp, .monotonic) != timestamp) {
+            uuid_v7_counter.store(@intCast(bun.fastRandom() % std.math.maxInt(u32)), .monotonic);
+        }
+
+        return uuid_v7_counter.fetchAdd(1, .monotonic) % 4096;
```

Comment:
Would it be possible to make `4096` configurable? Thinking about code like:
```js
const specificDate = Date.now();
for (let i = 0; i < 10_000; i ++) {
    Bun.randomUUIDv7(specificDate);
}
```

Reply:
No, it's the number of bits